styleguide/derek/pattern-components/resourceCenter-resource/refresh/_resourceCenter-resource.scss

```diff
@@ -69,7 +69,7 @@
 }
 
 .resource-logo {
-  background-image: url('../imgs/derek/rs-logo.svg');
+  background-image: url('../imgs/derek/rs_wordmark_white.svg');
   background-position: left 50%;
   background-repeat: no-repeat;
   background-size: 100% 79%;
```

Comment:
I think we need to adjust the size and position a little bit.  

This is what is currently on production: 
<img src="https://user-images.githubusercontent.com/5263371/49449958-a41d9e80-f7a1-11e8-94d2-a34a28090a41.png" width="300">


This is what you have: 
<img src="https://user-images.githubusercontent.com/5263371/49449806-5143e700-f7a1-11e8-873a-3265e8453647.png" width="300" />

Without fanatiguy it should probably align on the left.  Also the presence of fanatiguy made the text smaller previously.  Without it the text looks a little big. 


Let's try this: 
```
background-position: 39% 50%;
background-repeat: no-repeat;
background-size: 80% 63%;
```
This ends up looking like this: 
<img src="https://user-images.githubusercontent.com/5263371/49449780-45582500-f7a1-11e8-9fab-b337c999266e.png" width="300"/>